tests/integration/security/authorization_test.go

```diff
@@ -476,43 +476,129 @@ func TestAuthorization_IngressGateway(t *testing.T) {
 				Name     string
 				Host     string
 				Path     string
+				IP       string
 				WantCode int
 			}{
 				{
 					Name:     "allow www.company.com",
 					Host:     "www.company.com",
 					Path:     "/",
+					IP:       "172.16.0.1",
 					WantCode: 200,
 				},
 				{
 					Name:     "deny www.company.com/private",
 					Host:     "www.company.com",
 					Path:     "/private",
+					IP:       "172.16.0.1",
 					WantCode: 403,
 				},
 				{
 					Name:     "allow www.company.com/public",
 					Host:     "www.company.com",
 					Path:     "/public",
+					IP:       "172.16.0.1",
 					WantCode: 200,
 				},
 				{
 					Name:     "deny internal.company.com",
 					Host:     "internal.company.com",
 					Path:     "/",
+					IP:       "172.16.0.1",
 					WantCode: 403,
 				},
 				{
 					Name:     "deny internal.company.com/private",
 					Host:     "internal.company.com",
 					Path:     "/private",
+					IP:       "172.16.0.1",
 					WantCode: 403,
 				},
+				{
+					Name:     "allow remoteipblocks.company.com",
+					Host:     "www.company.com",
+					Path:     "/",
+					IP:       "172.16.0.1",
+					WantCode: 200,
+				},
+				{
```

Comment:
this one and the next one can just be removed, it's the same as the last one.

Reply:
I removed all three of the top 172.16.0.1 cases.  I originally put those cases there to test the host header separately to make sure it wouldn't confound the real tests, but that is still covered by the remaining tests.